content/docs/higher-order-components.md

```diff
@@ -1,56 +1,61 @@
 ---
 id: higher-order-components
-title: Higher-Order Components
+title: קומפוננטות מסדר גבוה יותר
 permalink: docs/higher-order-components.html
 ---
 
-A higher-order component (HOC) is an advanced technique in React for reusing component logic. HOCs are not part of the React API, per se. They are a pattern that emerges from React's compositional nature.
+קומפוננטה מסדר גבוה יותר (HOC) היא טכניקה מתקדמת של React שעוזרת למחזר קוד קומפוננטות.
+ה-HOC הן לא בדיוק חלק מהממשק של React, אלא תבנית עיצוב שהתפתחה מטבע ההרכבתיות של React.
+
+מבחינת היישום, **קומפוננטה מסדר גבוה יותר היא פונקציה שלוקחת קומפוננטה ומחזירה קומפוננטה אחרת**.
 
-Concretely, **a higher-order component is a function that takes a component and returns a new component.**
 
 ```js
 const EnhancedComponent = higherOrderComponent(WrappedComponent);
 ```
 
-Whereas a component transforms props into UI, a higher-order component transforms a component into another component.
+בשונה מקומפוננטה רגילה שמתרגמת מאפיינים לממשק משתמש, קומפוננטה מסדר גבוה יותר מתרגמת קומפוננטה לקומפוננטה אחרת.
 
-HOCs are common in third-party React libraries, such as Redux's [`connect`](https://github.com/reduxjs/react-redux/blob/master/docs/api/connect.md#connect) and Relay's [`createFragmentContainer`](http://facebook.github.io/relay/docs/en/fragment-container.html).
+ה-HOC שכיחות בספריות צד שלישי של React, כמו למשל ה- [`connect`](https://github.com/reduxjs/react-redux/blob/master/docs/api/connect.md#connect) של Redux וה- `createFragmentContainer`](http://facebook.github.io/relay/docs/en/fragment-container.html) של Relay.
```

Comment:
```suggestion
ה-HOCs שכיחות בספריות צד שלישי של React, כמו למשל ה- connect של Redux וה- [`createFragmentContainer`]( http://facebook.github.io/relay/docs/en/fragment-container.html) של Relay.
```

Reply:
Should I take off the link for "connect" or was that suggested by mistake on this line?